tests/test_parse_doc.m

```diff
@@ -0,0 +1,19 @@
+function test_parse_doc
+%TEST_PARSE_DOC Is the unit test corresponding to parse_doc.m
+%
+% DESCRIPTION
+%
+% That function makes sure that parse_doc is working properly and returns
+% the expected structure for the documentation.
+cd('../parser')
+docStruct = parse_doc('parse_doc.m');
+
+assert(isstruct(docStruct)) % Check if docStruct is indeed a structure
+
+% Make sure every input/output name matches a description
+if docStruct.inputs.names ~= ''
+assert(length(docStruct.inputs.names) == length(docStruct.inputs.description));
```

Comment:
You should add a descriptive message to the assert you're throwing. 
`assert(condition, message)`